### ISSUE TITLE
Google chrome not generating keys

### ISSUE THREAD

**Issue by ileathan** [linkeddata/profile-editor#20]
I go through the process to make a new account via databox.me but it only actually installs a certificate on firefox.

The account was created but the keys were not installed so i can never log back in and the recovery email doesnt seem to be working.

**Comment by deiu** [comment]
Yes, Chrome has disabled this behavior by default in the latest versions. You can manually enable it by click the green padlock next to the URL in the address bar, and allow `Key Generation`. Let me know if it worked.

**Comment by ileathan** [comment]
Oh thats annoying, thanks & hi :)

**Comment by ileathan** [comment]
Hey since its you.. >_> you think you can look at this for 30 seconds?

Go to https://solid.github.io/solid.js/demo/?
->
for the Profile URI enter: http://melvincarvalho.com/
->
You should get expected results but when you replace the uri with https://leathan.xyz/ it fails?

Im super confused because leathan.xyz is the same as melvincarvalho.com as far as I know. Dont spend any time on it but i think you will know whats wrong pretty quick since pretty much wrote it ^_^

**Comment by deiu** [comment]
There are a couple of issues with your server that I was able to easily spot.
1. It doesn't return `text/turtle` (which is standard for WebID profiles)
   `curl -H"Accept: text/turtle" https://leathan.xyz/`
2. The server doesn't support CORS:
   
   ```
   XMLHttpRequest cannot load https://leathan.xyz/. No 'Access-Control-Allow-Origin' 
   header is present on the requested resource. Origin 'https://solid.github.io' is 
   therefore not allowed access.
   ```

Melvin might be able to help you with that, if you're trying to follow his setup.

**Comment by ileathan** [comment]
I had this all working in the past and i more or less understand it so i thought quickly doing `wget --page-requisites` on melvins page would make for an easy boilerpoint since his page is working. 

Guess not, time to learn more :) Thanks for your help & work on webid stuff :)

**Comment by ileathan** [comment]
Sorry to reopen this but I have since spent my time reading about Solid. When I returned to try and make another profile from chrome I had to go to chromes content settings and add a key-generation exception for *.databox.me.
- When i initially load databox.me i am prompted for a username. When I enter my username it prompts me for my WebID. When i enter my WebID cert for https://leathan.xyz/#i it changes the button from "Check Avialability" to "Create Account".
- I then continue to press Create Account but afterwards it still generates a new WebID for me. Shouldnt we be able to log in with any ol WebID and use databox.me ldp servers?
- At first I thought it was a problem with my WebID but I have switched my WebID to text/turtle (cors enabled) and the problem persists. I cannot log in with my WebID and I can not send recovery emails to 1@leathan.xyz. I have burned through like 10 accounts now :/
- Adding a chrome check before key generation would be awesome and save a few acnts?

P.S. Is there a good tutorial/example for running an LDP server? They seem quite powerful for my project.

**Comment by bhaugen** [comment]
I got caught in this same trap. I created an exception for databox.me. How do I go about trying to create a certificate again? (I'm very new to the whole WebID thing.)

**Comment by ileathan** [comment]
Revisiting web-ids and i still cant seam to get anything to work for me locally, or on databox.me :(

I opened a new account god.databox.me :) but i think that one is burned too. when i click the login button i get an orange banner flash at the top of the screen saying "WebID Authentication failed"
